content/doc/developer/security/form-validation.adoc

```diff
@@ -107,3 +107,16 @@ public FormValidation doCheckFoo(@QueryParameter String value) {
   return FormValidation.ok();
 }
 ----
+
+Most form validation methods are invoked using HTTP GET, however, so the form needs to be adapted as well.
+
+The Jenkins form Jelly controls support the `checkMethod` attribute, which, if set to `post`, will result in form validation being invoked via HTTP POST.
+
+[source, xml]
+----
+<f:field name="foo">
```

Comment:
`f:entry field="foo"` I imagine :)

Reply:
Glad I got that wrong in the original and only copied and pasted here 😞

Reply:
And we didn't get that on the first PR, shame on us 👎